packages/start/entry-server/index.ts

```diff
@@ -21,8 +21,8 @@ export const render = (
   }
 ) =>
   composeMiddleware([
-    apiRoutes,
     inlineServerFunctions,
+    apiRoutes,
```

Comment:
Without this requests for server functions will be mistakenly handeld by wildcard routes (like 404) that now have a handler for all methods.